_redirects

```diff
@@ -6,3 +6,25 @@ layout: null
 /repl/build/:build /7/ 200
 # Previous versions of the REPL
 /repl/version/:version /7/ 200
+
+# Ported from redirect_from
+
+/docs/using-babel/                  /docs/setup/
+/plugins.html                       /docs/setup/
+/docs/using-6to5/                   /docs/setup/
+/docs/usage/browser/                /docs/setup/
+/browser.html                       /docs/setup/
+/docs/core-packages/babel-register/ /docs/usage/babel-register/
+/docs/usage/require/                /docs/usage/babel-register/
+/docs/usage/options/                /docs/usage/api/
+/docs/advanced/plugins/             /docs/plugins/
+/docs/usage/modules/                /docs/plugins/
+/caveats.html                       /docs/usage/caveats/
+/usage.html                         /docs/usage/cli/
+/docs/faq/                          /faq/
+/php.html                           /php/
```

Comment:
Maybe add `/repl.html` too, We removed that as it was causing issues with the site build, but the redirect might still be useful.